R/heatmaply.R

```diff
@@ -563,7 +577,21 @@ heatmaply.default <- function(x,
   }
 
   # this is to fix the error: "argument * matches multiple formal arguments"
-  if (!missing(col)) colors <- col
+  if (!is.null(col)) colors <- col
+
+  if (is.null(scale_fill_gradient_fun)) {
+    if (node_type == "heatmap") {
```

Comment:
Both scale_fill_gradient_fun seem identical to me, so why the if (node_type == "heatmap")?

Reply:
`scale_fill_gradientn` and `scale_color_gradientn`

Reply:
now I see, thanks :)